### PR TITLE
Parameters and migration guide for GPS waypoint follower

### PR DESCRIPTION
**Pull request by pepisg**
Add new parameter to nav2_waypoint_follower configuration's guide for gps waypoint follower

### CODE REVIEW

File: migration/Galactic.rst [hunk 1]
```diff
@@ -226,14 +226,18 @@ Added Smoother Task Server
...
+`This PR 2627 <https://github.com/ros-planning/navigation2/pull/2814>`_  adds the `follow_gps_waypoints` action server in `nav2_waypoint_follower`. This server accepts a set of GPS goals instead of cartesian goals and provides all the other functionalities available on `nav2_waypoint_follower`. A new tutorial demonstrating its functionality was also added on `PR 47 on navigation2_tutorials <https://github.com/ros-planning/navigation2_tutorials/pull/47>`_
 
 Removed Use Approach Velocity Scaling Param in RPP
 **************************************************
 
 The parameter ``use_approach_linear_velocity_scaling`` is removed in favor of always on to help in smooth transitions to the goal. `This PR 2701 <https://github.com/ros-planning/navigation2/pull/2701>`_ implements.
 
 Refactored AMCL motion models as plugins
-****************************************
+****************************************nav2_waypoint_follower
```
[the remark this answers is not in the document]
Comment (pepisg) [reply migration/Galactic.rst:240]: Yes, I'm really sorry. Fixing now

File: migration/Galactic.rst [hunk 2]
```diff
@@ -226,14 +226,18 @@ Added Smoother Task Server
 
 A new task server was added which loads smoother plugins and executes them to improve quality of an existing planned path. Smoothing action can be called from a behavior tree using SmoothPath action node. `PR 2569 <https://github.com/ros-planning/navigation2/pull/2569>`_ implements and `PR 2875 <https://github.com/ros-planning/navigation2/pull/2875>`_ adds in the first of the plugins using it with a simple smoother. Other smoothers are in development and will be added in the future.
 
+Added GPS Waypoint Follower Server
+**********************************
```
[the remark this answers is not in the document]
Comment (pepisg) [reply migration/Galactic.rst]: Done

File: migration/Galactic.rst [hunk 3]
```diff
@@ -226,14 +226,18 @@ Added Smoother Task Server
...
+
+`This PR 2627 <https://github.com/ros-planning/navigation2/pull/2814>`_  adds the `follow_gps_waypoints` action server in `nav2_waypoint_follower`. This server accepts a set of GPS goals instead of cartesian goals and provides all the other functionalities available on `nav2_waypoint_follower`. A new tutorial demonstrating its functionality was also added on `PR 47 on navigation2_tutorials <https://github.com/ros-planning/navigation2_tutorials/pull/47>`_
```
[the remark this answers is not in the document]
Comment (pepisg) [reply migration/Galactic.rst]: Done

**Review by SteveMacenski — APPROVED** [review]

**Comment by SteveMacenski** [comment]
New PR includes

**Pull request closed without merging**